syntax/markdown.vim

```diff
@@ -151,11 +151,11 @@ endif
 
 " Strike through
 if get(g:, 'vim_markdown_strikethrough', 0)
-    syn region mkdStrike matchgroup=mkdStrike start="\%(\~\~\)"    end="\%(\~\~\)"
+    execute 'syn region mkdStrike matchgroup=htmlStrike start="\%(\~\~\)" end="\%(\~\~\)"' . s:concealends
```

Comment:
I don't actually properly understand what `matchgroup` does here, so I don't know if this is correct – the end delimiter isn't concealed if I use `mkdStrike`, but any other group name seems to work fine.